test/junit/scala/collection/concurrent/ConcurrentMapTester.scala

```diff
@@ -0,0 +1,41 @@
+package scala.collection.concurrent
+
+import java.util.concurrent.Executors
+import java.util.concurrent.atomic.AtomicReference
+import scala.collection.immutable.Queue
+import scala.concurrent.duration.TimeUnit
+
+class ConcurrentMapTester[K, V](map: Map[K, V]) {
+  private[this] val tasks = new AtomicReference[Seq[Runnable]](Queue.empty)
```

Comment:
There's no concurrent access to `tasks`, is the `AtomicReference` future-proofing for tests that would do so?

Reply:
honestly, it's an artifact of a substantially different tester design. I'll try to simplify it

Reply:
now a `ConcurrentMapTester` can be used more than once (i.e. `runTasks` then check the results, `runTasks` again, etc.)